examples/form/src/App.js

```diff
@@ -99,22 +111,33 @@ class App extends Component {
                 label="Send Form" />
             </div>
             <div>
-              <span>Clik me to change first checkbox</span>
+              <span>Click me to change first checkbox</span>
               <Button
                 onClick={this.onChangeCheckbox1}
                 color="green"
                 label="Change first checkbox" />
             </div>
             <div>
-              <span>Clik me to change this checkbox</span>
+              <span>Click me to change this checkbox</span>
```

Comment:
I would change this span to something related to the `Textbox`
Like `Type something into the textbox`.